reporting_app/templates/proc_report.html

```diff
@@ -1,57 +1,106 @@
+{% macro proc_display(proc) -%}
+{# args:
+    proc (dict) - analysis_driver_proc from the Rest API.
+#}
+
+    <ul>
+        <li>Started: {{ proc._created }}</li>
+        <li>Ended:
+            {% if proc.end_date %}
+            {{ proc.end_date }}
+            {% else %}
+            running with pid: {{ proc.pid }}
+            {% endif %}
+        </li>
+        <li>
+            Status:
+            {% if proc.status %}
+            {{ proc.status }}
+            {% else %}
+            inactive
+            {% endif %}
+        </li>
+
+{%- endmacro %}
+
+
+{% macro proc_timeline(proc) -%}
+{# args:
+    proc (dict) - analysis_driver_proc from the Rest API.
+#}
+
+    <script src="../static/node_modules/vis/dist/vis.js"></script>
+    <script src="../static/node_modules/moment/moment.js"></script>
+    <link href="../static/node_modules/vis/dist/vis.css" rel="stylesheet" type="text/css" />
+    <div id="{{ proc.proc_id }}"></div>
+
+
+    <script type="text/javascript">
+      var container = document.getElementById('{{ proc.proc_id }}');
+
+    var stages = {{ proc.stages|list|safe }};
+
+    var stage_names = stages.map(function(p){
+        if (p.date_finished){
+        return {
+        content: p.stage_name,
+        'start': moment(p.date_started, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'),
+        'end': moment(p.date_finished, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS')}
+
+        } else {
+        return {
+        'start': moment(p.date_started, "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'),
+        'style': "color: red; background-color: pink;"
+        }
+    }});
+
+
+    var items = new vis.DataSet(stage_names);
+    startDate = (moment('{{ proc._created }}', "DD-MM-YYYY HH:mm:SS").format('YYYY-MM-DD HH:mm:SS'))
+    startMargin = moment(startDate).subtract(1, 'days')
+    endMargin = moment(startDate).add(6, 'days')
+    var options = {
+    min: startMargin,
+    max: endMargin
+    }
+    var timeline = new vis.Timeline(container, items, options);
+
+
+
+
+    </script>
+
+
+{%- endmacro %}
+
+
+
 {% macro proc_report(procs) -%}
 {# args:
     procs (list[dict]) - analysis_driver_procs from the Rest API.
 #}
+    {% for proc in procs[1:] %}
```

Comment:
This should be in the tabs in line 97 and 101.
And it is missing the first element of the list.